packages/esm-patient-orders-app/src/components/test-order.component.tsx

```diff
@@ -24,91 +24,111 @@ interface TestOrderProps {
 const TestOrder: React.FC<TestOrderProps> = ({ testOrder }) => {
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
-  const { concept, isLoading: isLoadingTestConcepts } = useOrderConceptByUuid(testOrder.concept.uuid);
-  const { encounter, isLoading: isLoadingResult } = useLabEncounter(testOrder.encounter.uuid);
+  const { concept, isLoading: isLoadingTestConcepts } = useOrderConceptByUuid(testOrder?.concept?.uuid);
+  const { encounter, isLoading: isLoadingResult } = useLabEncounter(testOrder?.encounter?.uuid);
 
-  const tableHeaders: Array<{ key: string; header: string }> = [
-    {
-      key: 'testType',
-      header: testOrder.orderType.display,
-    },
-    {
-      key: 'result',
-      header: t('result', 'Result'),
-    },
-    {
-      key: 'normalRange',
-      header: t('normalRange', 'Normal range'),
-    },
-  ];
+  const tableHeaders = useMemo(
+    () => [
+      {
+        key: 'testType',
+        header: testOrder?.orderType?.display || t('test', 'Test'),
```

Comment:
`testOrder.orderType.display` was previously assumed to be correct. If that assumption doesn't hold, that this shouldn't render anything. Just putting "Test" there is papering over an issue.

Reply:
I also don't know under what circumstance you would actually run into this.